scripts/build.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+#
+# This script builds the networkservicemesh
+#
+
+go get -u github.com/golang/protobuf/protoc-gen-go
+go generate ./...
+go install ./...
+go test ./...
```

Comment:
Can you add -race to this?

Race doesn't provide false positives, so it should be safe to add.

Reply:
Will do!

Reply:
@fkautz See here as to why "go test -race" won't work with alpine linux:

https://github.com/golang/go/issues/14481